frontend-react/src/pages/submissions/SubmissionTable.tsx

```diff
@@ -153,7 +174,9 @@ function SubmissionTable() {
             fallbackComponent={() => <ErrorPage type="message" />}
         >
             <Suspense fallback={<Spinner />}>
-                {isNumberedPaginationOn && <div>TK</div>}
+                {isNumberedPaginationOn && (
+                    <SubmissionTableWithNumberedPagination />
```

Comment:
I'd like to maintain the pattern of using props to determine additional table component functionality, as seen by passing cursorManager in now (and very much _not_ seen in how we handle the filter UI right now). The logic to show the different pagination UI can exist inside of the `Table` component [around here](https://github.com/CDCgov/prime-reportstream/blob/3c043d3dc92042431b6645c67e646921d314520b/frontend-react/src/components/Table/Table.tsx#L485-L487).

Reply:
Yep, that's still my plan. The rendering of `Table` is factored out into the `SubmissionTableContent` component, which will be used by both feature flag branches. The reason `SubmissionTableWithCursorManager` and `SubmissionTableWithNumberedPagination` are separate components is because of how differently they need to fetch results -- `useResource` vs `usePagination` -- and the fact that you can't conditionally call hooks.

The integration with `usePagination` is coming in #5577, but you can see how different the pre-`Table` code is here: https://github.com/CDCgov/prime-reportstream/pull/5496/files#diff-49bba1f24f8c082ab6d04fb65f3fd2e9ea5d3e4639716282aca973084ded93a9R34

Reply:
Ahhh, okay, thanks for the explanation!